design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -144,6 +144,287 @@ Note:
   </ros2_control>
 ```
 
+```xml
+  <ros2_control name="12DOF_Legged_System_Robot_MultiInterface" type="system">
```

Comment:
Please use capitalized CamelCase in the names. We will also change this in #25 

e.g. "12DoFLeggedSystemRobotMultiInterface"